### PR TITLE
New monitoring solution.

### LINKED ISSUE #4: Add a client-perf mode
Add a mode to the clients (via url param, perhaps) that will log a warning if a client draw is too slow (> 16ms).

### PR DESCRIPTION
**Pull request by applmak**
Renders monitors above the client surfaces. Very shiny. Enable with
--enable_monitoring on the server cmd-line & monitor=1 in the client.

Fixes #4.

### CODE REVIEW

File: client/style.css [hunk 1]
```diff
@@ -81,3 +81,48 @@ div[debug] {
...
+    float: left;
+}
+.label.late {
+    float: right;
+}
+.label.now {
+  position: absolute;
+  padding-left: 10px;
+  left: 50%;
+}
```
Comment (plural) [inline client/style.css]: add newline

File: server/monitoring/monitor.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    enabled = true;
+    monitoringSockets.forEach(sendCurrentState);
+  },
+  update: function(change) {
+    if (enabled) {
+      Object.assign(currentStatus, change);
+      monitoringSockets.forEach(socket => socket.emit('monitor', change));
+    }
+  }
+};
```
Comment (plural) [inline server/monitoring/monitor.js]: add newline.  This is true on several files here.

File: client/style.css (hunk 1 above)
Comment (applmak) [inline client/style.css]: Done.

**Comment by applmak** [comment]
Also, guarded all monitor calls.

File: server/server.js [hunk 3]
```diff
@@ -135,3 +137,6 @@ network.on('lost-client', function(id) {
   manager.dropClient(id);
 });
 
+if (flags.enable_monitoring) {
+  monitor.enable();
+}
```
Comment (plural) [inline server/server.js]: add newline.

**Pull request merged**